src/stores/CartContext/useCartContextSelector.ts

```diff
@@ -0,0 +1,11 @@
+import { useContext } from 'react';
+
+import { CartApiContext, CartContext } from './cartStore';
+
+export function useCartSelector() {
+  return useContext(CartContext);
+}
```

Comment:
```suggestion
export function useCartSelector() {
  if (!CartContext) {
    throw new Error("useCartSelector를 CartContextProvider 밖에서 실행했네요.");
  }

  return useContext(CartContext);
}
```

이런식의 에러 처리는 어떨까요?
context를 사용할 때 예외처리를 해주지 않으면 provider 밖에서 가끔 사용할 때 에러를 명시적으로 해주면 디버깅하기 쉬울 것 같아서요ㅎㅎ

Reply:
감사합니다! 에러처리랑 에러바운더리를 좀 보강해야겠습니다 👍